### DISCUSSION TITLE
Contact information required after install

### DISCUSSION THREAD

**Discussion by mzb2xeo** [rubyforgood/pet-rescue#1307] [General]
I would really like to try this. Is there any additional /updated documentation for installing and using it?

**Comment by kasugaijin** [comment]
@mzb2xeo sorry I didn’t see this discussion! I don’t get notified for new ones. I see you also posted an issue for better documentation. What step are you stuck on? Better to respond on the issue as then I will be notified.

**Reply by mzb2xeo** [reply]
@kasugaijin Hi!

I followed the [Docker installation instructions](https://github.com/rubyforgood/homeward-tails?tab=readme-ov-file#docker) to set up both Docker Desktop and Ubuntu with Docker. Unfortunately, in both cases, I ended up with a demo site that has issues with default login credentials and does not appear to allow editing.  

Any guidance would be appreciated.

**Reply by kasugaijin** [reply]
So the app runs in docker and you can visit it on localhost? Have you tried attaching to the rails shell to view the logs for errors?
